apisix/secret/vault.lua

```diff
@@ -53,10 +54,15 @@ local function make_request_to_vault(conf, method, key, data)
     local req_addr = conf.uri .. norm_path("/v1/"
                 .. conf.prefix .. "/" .. key)
 
+    local token, _ = env.fetch_by_uri(conf.token)
```

Comment:
Just use `core.env.fetch_by_uri`

Reply:
I think it's okay to import using `require` and use because if in the future we need to use the env module again in this file then we won't have to re-import using `require`.

Also, at that time in the future, we will not have to replace `core.env.fetch_by_uri` with `env.fetch_by_uri`. WDYT?